safe-global/web-core
# fix: tx details header

## What it solves
The header of tx details was taking too much space and displaying a wrong label.

Resolves #813 

## How this PR fixes it
- Displays a simple label for transaction details.
- Adds a hint which tx needs to be executed before the displayed transaction to explain why the execute button is disabled.

## How to test it
- Queue multiple transactions.
- Click share button on one of the transactions with high nonce and open the copied link
- Witness that a hint is displayed that another tx needs to be executed first.

## Analytics changes
None

## Screenshots
![localhost_3000_gor_0x954cD69f0E902439f99156e3eeDA080752c08401_transactions_tx_id=multisig_0x954cD69f0E902439f99156e3eeDA080752c08401_0x9a5eb65ec51701fe1178a8ff40d43ad9deed0b7cf3c3d6633e9b804edf896469](https://user-images.githubusercontent.com/2670790/194117333-8ec0b0c1-97a7-4758-84cd-5588ab764b79.png)
